agent/Dockerfile

```
@@ -71,7 +71,15 @@ RUN golangci-lint run --timeout=5m
 # =============================================================================
 FROM go-base AS tester
 
-RUN go test ./... -v
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)